SAMtemplates/state_machines/GetMyPrescriptionsStateMachine.asl.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Comment": "Get My Prescriptions State Machine",
+  "StartAt": "Get My Prescriptions",
+  "States": {
+    "Get My Prescriptions": {
+      "Type": "Task",
+      "Resource": "arn:aws:states:::lambda:invoke",
+      "Parameters": {
+        "Payload.$": "$",
+        "FunctionName": "${GetMyPrescriptionsFunctionArn}"
+      },
+      "End": true
+    }
```

Comment:
I know Phil is probably putting it in as well, but we should have a catch all step that returns a FHIR error otherwise if there is an unexpected error you get a 200 response back with no body